### PR TITLE
[WIP] What's New

### PR DESCRIPTION
**Pull request by timshell**
Re: #94 

Not sure what level of granularity we're looking for. Also don't know contributions besides general NeuroVis and NeuroPop before I started

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Changelog
+~~~~~~~~~
+
+    - Add NeuroVis class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
+    - Add NeuroPop class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
+    - Add PopVis class by `Mayank Agrawal`_
+    - Add STRF class by `Pavan Ramkumar`_
+    - Add Sphinx Documentation `Mayank Agrawal`_
+    - Add Travis CI by `Mayank Agrawal`_
+    - Add Circle CI by `Mainak Jas`_ and `Mayank Agrawal`_
```
Comment (jasmainak) [inline doc/whats_new.rst]: I wouldn't put this in the changelog :) Just new features etc. Users don't care about Circle CI

File: doc/whats_new.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - ...
+
+
+Version 0.1
+-----------
+
+Changelog
+~~~~~~~~~
+
+    - Add NeuroVis class by `Hugo Fernandes`_ and `Pavan Ramkumar`_
```
Comment (jasmainak) [inline doc/whats_new.rst]: can you say ``:class:`pyglmnet.NeuroVis``, like here: https://github.com/mne-tools/mne-python/blob/master/doc/whats_new.rst

**Comment by timshell** [comment]
should be good to go @pavanramkumar

**Comment by pavanramkumar** [comment]
@timshell do you want to rebuild?

**Pull request merged**